easybuild/easyconfigs/m/meshtool/meshtool-20250401-GCCcore-13.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'meshtool'
+version = '20250401'
+
+homepage = 'https://bitbucket.org/aneic/meshtool'
+description = """Meshtool is a comand-line tool written in C++.
+ It is designed to apply various manipulations to volumetric meshes."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'openmp': True}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
```

```suggestion
    'filename': '%(name)s-%(version)s.tar.xz',
```

We need to use XZ extension in order to have reproducible checksums of git tag/commit downloads